### PR TITLE
feat: add chart legends, make titles smaller

### PR DESCRIPTION
**Pull request by diegofigs**

### CODE REVIEW

**Comment by DonDuala** [comment]
1. Add a blue dash and call it "Orders" 
2. Make the squares into circles
image

File: app/src/pages/liquidity/liquidity.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React, { useEffect, useState } from 'react';
```
Comment (DonDuala) [inline app/src/pages/liquidity/liquidity.tsx:1]: I had to add a line here to get this to work:
import Typography from '@material-ui/core/Typography';

Comment (brittcyr) [reply app/src/pages/liquidity/liquidity.tsx:1]: 2023-01-23T19:20:36.369Z [INFO]: [eslint]
                                 src/pages/liquidity/liquidity.tsx
                                 Line 81:8:  'Typography' is not defined  react/jsx-no-undef
                                 Line 84:8:  'Typography' is not defined  react/jsx-no-undef
                                 Search for the keywords to learn more about each error.


Build is failing for that reason.

**Review by brittcyr — APPROVED** [review]
looks fine
https://feat-add-chart-legends.d2wsaca7g6d2id.amplifyapp.com/

**Review by DonDuala — APPROVED** [review]

**Pull request merged**